### PR TITLE
Updating and Maintaining Pastebin Checker

### PR DESCRIPTION
**Pull request by LivelyCarpet87**
Changes:
- Removed the 1-minute interval script because it is a repeat of the 3-minute interval script and is more likely to get blocked by pastebin.com for the high traffic.
- Updated the 3-minute interval script to:
  - properly migrate to the newer API parameters such as `shell`
  - use the xbar vars for configuration
  - general bug fixes

### CODE REVIEW

**Comment by LivelyCarpet87** [comment]
Could someone please quickly skim over the changes and approve the update? The app right now is slightly broken and needs these changes to be compatible with the newer versions of xbar. Thanks.

File: Dev/Pastebin/pastebin.3m.sh [hunk 1]
```diff
@@ -2,31 +2,33 @@
...
+# <xbar.version>v2.0</xbar.version>
 # <xbar.author>Tyllis Xu</xbar.author>
 # <xbar.author.github>livelycarpet87</xbar.author.github>
 # <xbar.desc>Uses a set of Pastebin API keys to check for pastes created by the user. It will provide links to all the pastes it finds by the user. </xbar.desc>
-# <xbar.image>https://i.ibb.co/cym797V/Pastebin2.png</xbar.image>
-# <xbar.abouturl>https://github.com/LivelyCarpet87/BitBar-Pastebin#bitbar-pastebin</xbar.abouturl>
+# <xbar.image>https://tyllisxu.com/assets/img/portfolio/BitBar-Pastebin/pastebin.png</xbar.image>
+# <xbar.abouturl>https://github.com/LivelyCarpet87/xbar-Pastebin#xbar-pastebin</xbar.abouturl>
+# <xbar.dependencies>bash,curl</xbar.dependencies>
+# <xbar.var>string(VAR_VAR_DEV_KEY=""): Your Pastebin API developer key. This can be obtained at pastebin.com/doc_api after logging in.</xbar.var>
```
Comment (matryer) [inline Dev/Pastebin/pastebin.3m.sh:12]: Is there a reason why you have `VAR_VAR_` here?

**Comment by matryer** [comment]
Thanks for updating this @LivelyCarpet87 - I left a question about the var names.

**Comment by LivelyCarpet87** [comment]
Terribly sorry, I must have missed a variable name when mass renaming. I'll fix it now.

File: Dev/Pastebin/pastebin.3m.sh (hunk 1 above)
Comment (LivelyCarpet87) [reply Dev/Pastebin/pastebin.3m.sh:12]: This was an error probably created when I was replacing variable names. Now fixed.

**Comment by LivelyCarpet87** [comment]
Just a reminder that the variable names were fixed. Sorry for the previous delay, I forgot to push changes.

**Comment by LivelyCarpet87** [comment]
Bumping this thread again before it disappears into the void of unfixed pull requests. The previous variable name problem was fixed.

**Review by sprak3000 — APPROVED** [review]

**Pull request merged**